pax-example/.pax/reexports.partial.rs

```diff
@@ -1,8 +1,15 @@
 pub mod pax_reexports { 
-	pub use crate::PaxExample;
+	pub use crate::Example;
+	pub mod fireworks {
```

Comment:
note: updated the reexports generation logic to enabled crate-local modules like `fireworks` here — this was necessary to import multiple Pax components in a single project (see the primitive router in `pax-example/src/lib.rs`)